src/starknet-wrappers.ts

```diff
@@ -28,6 +31,7 @@ interface DeclareWrapperOptions {
     token?: string;
     sender?: string;
     nonce?: string;
+    version?: number;
```

Comment:
Where is this used and why is it optional?

Reply:
Great catch! https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/328/commits/719d60947d3300cd0af7b2534eb8041fa5ceaa33